.gitignore

```diff
@@ -11,6 +11,7 @@ public
 node_modules/
 .tmuxp.*
 .idea
```

Comment:
.tmp should go in here as well

```suggestion
.idea
.tmp
```

Reply:
Actually not needed, I just noticed that you're putting a .gitignore in the .tmp folder, which is a great idea.